src/js/tabs/power.js

```diff
@@ -211,19 +211,19 @@ TABS.power.initialize = function (callback) {
         $("#calibrationmanagerconfirmcontent").hide();
 
        // battery
-        var template = $('#tab-power-templates .battery-state .battery-state');
-        var destination = $('.tab-power .battery-state');
-        var element = template.clone();
+        let template = $('#tab-power-templates .battery-state .battery-state');
+        let destination = $('.tab-power .battery-state');
+        let element = template.clone();
         $(element).find('.connection-state').attr('id', 'battery-connection-state');
         $(element).find('.voltage').attr('id', 'battery-voltage');
         $(element).find('.mah-drawn').attr('id', 'battery-mah-drawn');
         $(element).find('.amperage').attr('id', 'battery-amperage');
 
         destination.append(element.children());
 
-        var template = $('#tab-power-templates .battery-configuration');
-        var destination = $('.tab-power .battery .configuration');
-        var element = template.clone();
+        template = $('#tab-power-templates .battery-configuration');
+        destination = $('.tab-power .battery .configuration');
```

Comment:
The same for the `destination` variable that for the `template`, is better to have it's own variable at each use.

Reply:
Did a rebase, much more clear now.